tests/integration/targets/ntnx_protection_policies_v2/vars/main.yml

```diff
@@ -0,0 +1,301 @@
+replication_location_1:
```

Comment:
Can you please add some explaination about what this vars does and what was your intention behind this ?

Reply:
Also mention in protection_policies.yml that how you are consuming this there.